server/journeys/referral/details/details.cy.ts

```diff
@@ -151,6 +154,8 @@ context('Make a Referral Journey', () => {
     cy.findByRole('textbox', { name: /minute/i }).type('23')
```

Comment:
Might be worth using a different value for hour and minute just to be sure the right one has been retained in each case

Reply:
Done :)